smartBugs.py

```diff
@@ -85,7 +83,7 @@ def exec_cmd(args: argparse.Namespace):
                     # local copy does not exist; we need to clone it
                     print('\x1b[1;37m' + "%s is a remote dataset. Do you want to create a local copy? [Y/n] " % base_name + '\x1b[0m')
                     answer = input()
-                    if answer in ['y', 'Y', 'Yes', 'YES', 'Ye', 'YE']:
+                    if answer in ['y', 'Y', 'yes', 'Yes', 'YES', '']:
```

Comment:
Does the empty string ('') make sense in this list?

Reply:
It represents the default option (yes). If you press enter, it's the same as yes.

Reply:
Would it work if someone would write `yEs`? if so, perhaps the better is:

`answer.lower() in ['yes', 'y', '']`

Reply:
Done.